### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by evesnow91**

### CODE REVIEW

**Comment by anton-trunov** [comment]
Should we merge this?

**Comment by jjcnn** [comment]
@evesnow91 : Is this PR still valid, or have the examples changed since?

**Comment by evesnow91** [comment]
Yes, it is still valid. Examples have not changed

File: docs/source/scilla-in-depth.rst [hunk 1]
```diff
@@ -2401,7 +2401,8 @@ gas already consumed).
 
 The total number of executions that can happen in a single chain call
 sequence (starting from first execution that was triggered from a non-contract
-account) is currently 10. This is subject to revision.
+account) is currently set at ``10`` edges. Edges includes both breath and depth.
```
Comment (anton-trunov) [inline docs/source/scilla-in-depth.rst:2404]:
```suggestion
account) is currently set at ``10`` edges. Edges includes both breadth  and depth. 
```

File: docs/source/scilla-in-depth.rst [hunk 2]
```diff
@@ -2401,7 +2401,8 @@ gas already consumed).
 
 The total number of executions that can happen in a single chain call
 sequence (starting from first execution that was triggered from a non-contract
-account) is currently 10. This is subject to revision.
+account) is currently set at ``10`` edges. Edges includes both breath and depth. 
+The number may be subjected to revision in future.
```
Comment (anton-trunov) [inline docs/source/scilla-in-depth.rst:2405]:
```suggestion
The number may be subjected to revision in the future.
```

File: docs/source/scilla-trial.rst [hunk 3]
```diff
@@ -75,3 +78,7 @@ Savant IDE comes with the following sample smart contracts written in Scilla:
 + **SchnorrTest** : A sample contract to test the generation of a Schnorr 
   public/private keypairs, signing of a ``msg`` with the private keys,
   and verification of the signature.
+
++ **ECDSATest** : A sample contract to test the generation of a ECDSA 
+  public/private keypairs, signing of a ``msg`` with the private keys,
```
Comment (anton-trunov) [inline docs/source/scilla-trial.rst:83]: Maybe `msg` should be replaced with "message"?

**Pull request merged**